packages/react/src/components/DataGrid/DataList.spec.tsx

```diff
@@ -0,0 +1,138 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import React, { useState } from "react";
+import { faker } from "@faker-js/faker";
+import { getAllByRole, getByRole, queryAllByRole } from "@testing-library/dom";
+import { expect } from "vitest";
+import userEvent from "@testing-library/user-event";
+import { expectPaginationList } from "components/Pagination/utils";
+import { CunninghamProvider } from "components/Provider";
+import { DataList } from "components/DataGrid/DataList";
+import { sleep } from "utils";
+import { Row } from "components/DataGrid/index";
+
+describe("<DataList/>", () => {
+  it("should render a DataList", async () => {
+    const rows = Array.from(Array(3)).map((_value, index) => ({
+      id: `list key for element ${index}`,
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+    }));
+    render(
+      <CunninghamProvider>
+        <DataList
+          columns={[{ field: "firstName" }, { field: "lastName" }]}
+          rows={rows}
+        />
+      </CunninghamProvider>
+    );
+
+    const table = screen.getByRole("table");
+    expect(queryAllByRole(table, "columnheader")).toEqual([]);
+
+    rows.forEach((row) => {
+      const element = screen.getByTestId(row.id);
+      const tds = getAllByRole(element, "cell");
+      expect(tds.length).toBe(2);
+      expect(tds[0].textContent).toEqual(row.firstName);
+      expect(tds[1].textContent).toEqual(row.lastName);
+    });
+  });
+  it("should render a DataList with working rows selection", async () => {
```

Comment:
IMO this test is redundant as it is already test in `SimpleDataGrid.spec.tsx`, WDYT?

Reply:
I took `SimpleDataGrid` tests as a template.
I thought that loading was already tested twice in `DataGrid` and `SimpleDataGrid`, so it should also be tested here.

If we refactor this `DataList` not to configure `DataGrid` but to be used in it, it can be convenient to have duplicated tests. 

I may as well remove these redundant tests, we can easily add them later if we refactor.